src/tests/test_main.py

```diff
@@ -187,6 +189,18 @@ def test_expand_uris(self):
         ]
         self.assertEqual(main.expand_uris(msid, given), expected)
 
+    def test_expand_inline_graphics(self):
```

Comment:
If the inline-graphics load from the CDN rather than the IIIF server, then that is much simpler. Is that the same for video stills, or are those still coming through IIIIF?